amppkg.example.toml

```diff
@@ -170,3 +188,75 @@ ForwardedRequestHeaders = []
   # Domain = "www.corp.amppackageexample.com"
   # PathRE = "/world/.*"
   # QueryRE = ""
+
+# ACME is a protocol that allows for automatic renewal of certificates. Amp packager uses an ACME library
+# https://github.com/go-acme/lego to handle certificate renewal.
+#
+# For the full ACME spec, see:
+# 	https://tools.ietf.org/html/draft-ietf-acme-acme-02
+# 	https://ietf-wg-acme.github.io/acme/draft-ietf-acme-acme.html
+#
+# ACMEConfig only needs to be present in the toml file if 'autorenewcert' command line flag was turned on.
+# If the flag is on, at least one of ACMEConfig.Production or ACMEConfig.Development should be present.
+# Note that a recommended best practice for setting up the cert renewal that minimizes both cost and bombarding
+# your Certificate Authority with requests is that for a multi-instance setup of AMP packager, only one instance is
+# setup to do automatic cert renewals and the rest of the instances will just be configured to reload the fresh
+# certificate from disk when their in-memory copies expire.  This also implies that the cert paths configured above
+# in 'CertFile' and 'NewCertFile' are located on a shared filesystem accessible by all AMP packager instances.
+# [ACMEConfig]
+  # [ACMEConfig.Production]
+  # This config will be used if 'autorenewcert' is turned on and 'development' is turned off.
```

Comment:
Move this comment above [ACMEConfig.Production] to match the style in this file.

Reply:
Done.